modules/user_accounts/php/edit_user.class.inc

```diff
@@ -980,29 +980,11 @@ class Edit_User extends \NDB_Form
         if (empty($values['NA_Password'])
             && (!empty($values['Password_hash']) || !empty($values['__Confirm']))
         ) {
-            $isPasswordStrong = \User::isPasswordStrong(
-                $values['Password_hash'],
-                array(
-                 $values['__Confirm'],
-                 isset($values['UserID']) ? $values['UserID'] : $this->identifier,
-                 $values['Email'],
-                ),
-                array(
-                 '==',
-                 '!=',
-                 '!=',
-                )
-            );
-
-            // check password strength
-            if (!$isPasswordStrong) {
-                $errors['Password_Group']
-                    = 'The password is weak, or the passwords do not match';
-            } else {
-                // New password must be different than current one
-                if (!$this->_passwordChanged($DB, $values['Password_hash'])) {
-                    $errors['Password_Group'] = 'New and old passwords '
-                        . 'are identical: please choose another one';
+            $newPassword = $values['Password_hash'];
```

Comment:
Do we want to check with latest 5 password or some numbering scheme like
myPa$$word1
myPa$$word2
myPa$$word3
...

Reply:
We could do so. Usually this type of thing is done when there is a password expiration policy and people want to basically use the same password anyway. I can't remember exactly but I think our expiration window is fairly large so I don't think our users will be compelled to do this. I am planning another PR to remove password expiration altogether because it's not considered a good thing to have for exactly this reason.

Note also that this will require a database change as we only store the current `Password_hash` in the user table, not previous passwords.